lib/plugins/EPrints/Plugin/Import/PubRouter.pm

```diff
@@ -437,14 +403,7 @@ sub xml_to_epdata
                 	        $epdata->{date_type} = $date_type;
 				$date_set = 1;
 			}
-		}
-		
-		#always add date to Additional Information field
-		push @date_notes, "$date_type $date_string";		
-	}
-	if( scalar @date_notes > 0 )
-	{
-		$epdata->{note} = $plugin->appendToNote( $epdata->{note}, "History", join( "; ", @date_notes ) );	
+		}	
```

Comment:
In the above code I don't see purpose of local variable `$date_set` .

Reply:
$date_set was a left over from adding the date info to the notes field I think. I've now removed and added in some more comments - https://github.com/jisc-services/eprints-plugin-PubRouter-RIOXXplus-Connector/pull/5/commits/5d2140888ed57771e75f7fe23225da064962ed62